cinder/README.md

```diff
@@ -214,9 +216,26 @@ unity_storage_pool_names=
 
 ## Multiple Back-ends
 
-`cinder-dellemc-unity-config.yaml` and `cinder-dellemc-vnx-config.yaml` couldn't use to configure multiple backends.
+To configure multiple backends of two different storage types you can combine pass both environment files on the same deploy command as below.
 
-In order to configure multiple back-ends, we could use `ControllerExtraConfig` to set the configurations for all backends directly.
+`cinder-dellemc-unity-config.yaml` and `cinder-dellemc-vnx-config.yaml` can be used to configure both unity and vnx as multi-backends.
+
+```bash
+(undercloud) $ openstack overcloud deploy --templates \
+  -e /home/stack/templates/containers-prepare-parameter.yaml \
+  -e /home/stack/templates/custom-dellemc-container.yaml \
+  -e <other templates> \
+  -e /home/stack/templates/cinder-backend-dellemc-vnx.yaml
```

Comment:
"\" is needed at EOL.

Reply:
> "" is needed at EOL.

It is there, added the \ at EOL that was missing